phpcs.xml.dist

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0"?>
```

Comment:
This is just a common linting file that we use in all of our projects. It doesn't affect the behavior at all and is just a quality of life addition.